Product/Production/Common/Properties/src/main/resources/hibernate/admingui.hibernate.cfg.xml

```diff
@@ -21,7 +21,7 @@
             Switching to the classing translator so that our named query works.
         -->
         <property name="hibernate.query.factory_class">org.hibernate.hql.internal.classic.ClassicQueryTranslatorFactory</property>
-
+        <property name="hibernate.connection.datasource">java:comp/env/jdbc/adminguidb_datasource</property>
```

Comment:
Have you tested this without including "java:comp/env/" to see if it would work?